sky/spot/spot_utils.py

```diff
@@ -286,15 +295,20 @@ def stream_logs_by_id(job_id: int, follow: bool = True) -> str:
             spot_status = spot_state.get_status(job_id)
             if spot_status.is_terminal():
                 break
-            status_display.update(
-                _JOB_WAITING_STATUS_MESSAGE.format(status_str=''))
+            logger.info(
+                f'{colorama.Fore.YELLOW}The job is preempted or cancelled. '
+                'Waiting for its status to be updated.'
```

Comment:
Maybe we can leave this part out? It's in the spinner line.

Reply:
Good point! Removed it.